### DISCUSSION TITLE
Case à cocher qui est cochée automatiquement par défaut

### DISCUSSION THREAD

**Discussion by ageliinas** [MTESSDev/FRW#134] [Annonces]
Bonjour!

Nous avons un contexte particulier dans notre code ou nous aimerions que la case à cocher unique (checkbox) soit automatiquement cochée par défaut, car elle sera masquée à l'utilisateur du formulaire, mais utile pour la programmation TI. 

Merci,
Bonne journée!

**Comment by loima02** [comment]
Bonjour!

Avez-vous essayé avec le contrôle de type `hidden`?

Réf: https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionHidden

**Comment by ageliinas** [comment]
Bonjour, oui, pour la partie hidden il n'y a aucun problème. En gros, notre problème est quand l'utilisateur sélectionne le champs autre, une case à cocher demandant la vérification soit automatiquement coché et non vide. Vue qu'elle est hidden, on ne veut pas que l'utilisateur la coche,  il faut alors que ma case soit automatiquement coché par défaut.

**Comment by Hutson48** [comment]
Bonjour, si vous voulez vous une valeur par défaut vous devez faire un value. 
````yaml
- type: hidden
  name: hidden1
  value: true
````
N'hésitez pas à me revenir si vous avez des questions!
